sawmill-core/src/main/java/io/logz/sawmill/http/ExternalMappingResponse.java

```diff
@@ -0,0 +1,27 @@
+package io.logz.sawmill.http;
+
+import java.util.Map;
+
+public class ExternalMappingResponse {
+    private final Boolean modified;
```

Comment:
1. can be a simple `boolean`
2. not sure we even need this boolean - `isModified()` can just check if `mappings != null`